### ISSUE TITLE
The Title for the "Follow up: Fever: Location of temp Page" not showing

### ISSUE THREAD

**Issue by sthomas313-gatech** [Co-Epi/app-ios#290]
**Describe the bug**
As I was running thought accessibility stuff for the simulated iOS version of the app. I noticed the title for the page was missing. I went to see if the same issue occurred on my Android , it did not. So this is either a local issue on my version of the code or on the iOS side. 

**To Reproduce**
1. Select Fever option
2.Go through follow ups
3. Once you get to the where did you check temp page, the text does not appear.

**Screenshots**
<img width="386" alt="Screen Shot 2020-06-26 at 2 25 23 PM" src="https://user-images.githubusercontent.com/42740535/85889482-9089bc80-b7b9-11ea-8fa3-46d64a1dfbac.png">

**Comment by sthomas313-gatech** [comment]
This is a duplicate issue as #283
